src/Serilog/Events/ScalarValue.cs

```diff
@@ -19,6 +19,11 @@ namespace Serilog.Events;
 /// </summary>
 public class ScalarValue : LogEventPropertyValue
 {
+    /// <summary>
+    /// Scalar value representing <see langword="null"/>.
+    /// </summary>
+    public static ScalarValue Null { get; }= new(null);
```

Comment:
```suggestion
    public static ScalarValue Null { get; } = new(null);
```

🛃 nitpick - missing space